card/src/main/java/com/adyen/checkout/card/CardConfiguration.java

```diff
@@ -265,6 +283,26 @@ public Builder setEnvironment(@NonNull Environment builderEnvironment) {
         @NonNull
         public Builder setSupportedCardTypes(@NonNull CardType... supportCardTypes) {
             mBuilderSupportedCardTypes = Arrays.asList(supportCardTypes);
+            final List<CardBrand> cardBrands = new ArrayList<>();
+            for (CardType cardType : mBuilderSupportedCardTypes) {
+                cardBrands.add(new CardBrand(cardType));
+            }
+            mBuilderSupportedCardBrands.clear();
+            mBuilderSupportedCardBrands.addAll(cardBrands);
+            return this;
+        }
+
+
+        /**
+         * This is the same as setSupportedCardTypes just with another type {@link CardBrand}
```

Comment:
We should add the same docs from the original `setSupportedCardTypes` method here, and then explain that they are the same